File: app/src/main/java/remix/myplayer/ui/activity/AboutActivity.java
package remix.myplayer.ui.activity;

import android.content.pm.PackageInfo;
import android.content.pm.PackageManager;
import android.os.Bundle;
import android.widget.TextView;
import butterknife.BindView;
import butterknife.ButterKnife;
import remix.myplayer.App;
import remix.myplayer.R;

/**
 * Created by Remix on 2016/3/26.
 */
public class AboutActivity extends ToolbarActivity {

  @BindView(R.id.about_text)
  TextView mVersion;

  @Override
  protected void onCreate(Bundle savedInstanceState) {
    super.onCreate(savedInstanceState);
    setContentView(R.layout.activity_about);

    ButterKnife.bind(this);
    try {
      PackageManager pm = App.getContext().getPackageManager();
      PackageInfo pi = pm
          .getPackageInfo(App.getContext().getPackageName(), PackageManager.GET_ACTIVITIES);
      mVersion.setText("v" + pi.versionName);
    } catch (Exception ignored) {

    }

    setUpToolbar(getString(R.string.about));
  }

}

File: app/src/main/java/remix/myplayer/ui/activity/AboutActivity.kt
package remix.myplayer.ui.activity

import android.annotation.SuppressLint
import android.os.Bundle
import remix.myplayer.BuildConfig
import remix.myplayer.R
import remix.myplayer.databinding.ActivityAboutBinding
import remix.myplayer.misc.AppInfo
import remix.myplayer.theme.Theme
import remix.myplayer.theme.ThemeStore
import remix.myplayer.theme.TintHelper

class AboutActivity : ToolbarActivity() {
  private lateinit var binding: ActivityAboutBinding

  @SuppressLint("SetTextI18n")
  override fun onCreate(savedInstanceState: Bundle?) {
    super.onCreate(savedInstanceState)
    binding = ActivityAboutBinding.inflate(layoutInflater)
    setContentView(binding.root)

    binding.aboutText.text = "v${BuildConfig.VERSION_NAME}"
    binding.aboutText.setOnLongClickListener {
      Theme.getBaseDialog(this)
        .content(AppInfo.prettyPrinted)
        .positiveText(R.string.close)
        .build()
        .run {
          with(contentView ?: return@setOnLongClickListener false) {
            TintHelper.setTint(this, ThemeStore.accentColor, false)
            setTextIsSelectable(true)
          }
          show()
        }
      true
    }

    setUpToolbar(getString(R.string.about))
  }
}